RadarSDK/Radar.h

```diff
@@ -151,6 +152,13 @@ typedef void(^ _Nonnull RadarSearchPlacesCompletionHandler)(RadarStatus status,
  */
 typedef void(^ _Nonnull RadarSearchGeofencesCompletionHandler)(RadarStatus status, CLLocation * _Nullable location, NSArray<RadarGeofence *> * _Nullable geofences);
 
+/**
+Called when a point search request succeeds, fails, or times out. Receives the request status and, if successful, the location and an array of points sorted by distance.
+
+@see https://radar.io/documentation/api#search-geofences
+*/
+typedef void (^_Nonnull RadarSearchPointsCompletionHandler)(RadarStatus status, CLLocation *_Nullable location, NSArray<RadarPoint *> *_Nullable points);
```

Comment:
nit: other callbacks seem to have a space between `^` and `_` - this will likely get fixed / standardized with `.clang-format`

Reply:
yeah, this will get standardized in another PR